src/main/java/pages/MobileChapterPage.java

```diff
@@ -6,31 +6,30 @@
 import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
-
+import java.time.Duration;
 import java.util.List;
 
 public class MobileChapterPage {
-
     private WebDriver driver;
-    private int timeOut = 5;
-
-    private By mobilePageSlogan = By.xpath("//h1[contains(text(),'Mobiel App testen')]");
+    private int timeOut = 10;
     private By articleHeaderBlock = By.xpath("//h2[@class='h1 block__title']/a");
     private By contactUsButton = By.xpath("//a[contains(text(),'Neem contact op')]");
+    private By mobilePageSlogan = By.xpath("//h1[contains(text(),'Mobiel App testen')]");
+
 
     public MobileChapterPage(WebDriver driver){
         this.driver = driver;
         PageFactory.initElements(driver,this);
```

Comment:
Zoals je in de ContactUsPage al hebt laten zien kan deze initElements weggelaten worden.